r/R/dplyr-funcs-datetime.R

```diff
@@ -539,3 +595,21 @@ binding_as_date_numeric <- function(x, origin = "1970-01-01") {
 
   x
 }
+
+build_formats <- function(orders) {
+  year_chars <- sprintf("%%%s", c("y", "Y"))
+  month_chars <- sprintf("%%%s", c("m", "B", "b"))
+  day_chars <- sprintf("%%%s", "d")
```

Comment:
These each expand into `c("%y", "%Y")`, `c("%m", "%B", "%b")`, and `"%d"`, yeah? I know this is _slightly_ more DRY to do it with `sprintf()`, but it would be significantly shorter + less needing to think about it if we just wrote them out, no?

Reply:
Sure, there was a reason why I wrote them like that, but I don't remember it. Done.